hax/hax/motr/__init__.py

```diff
@@ -81,6 +84,9 @@ def start(self, rpc_endpoint: str, process: Fid, ha_service: Fid,
                 ' returned non-zero code (%s)', result)
             raise RuntimeError('Cannot start m0_halon_interface.'
                                'Please check Motr logs for more details.')
+        thread = RconfcStarter(self, self.consul_util)
+        thread.start()
+        return thread
```

Comment:
Please update the method signature to reflect the return type.

Reply:
Done.